front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -22,6 +25,35 @@ const ScenarioDescription = ({
 }: ScenarioDescriptionProps) => {
   const { t } = useTranslation('operationalStudies/scenario');
   const { openModal } = useModal();
+  const [isOpenedDescription, setIsOpenedDescription] = useState<boolean>(false);
+  const descriptionRef = useRef<HTMLDivElement | null>(null);
+  const ref = useRef<HTMLDivElement | null>(null);
+  const [isTooLongDescription, setIsTooLongDescription] = useState<boolean>(false);
+
+  const toggleDescription = () => {
+    setIsOpenedDescription(!isOpenedDescription);
+  };
+
+  const checkIfTooLong = () => {
+    if (ref && ref.current) {
+      ref.current.scrollHeight > 60
+        ? setIsTooLongDescription(true)
+        : setIsTooLongDescription(false);
```

Comment:
This could be simplified with a single call to `setIsTooLongDescription()`, I think:

```js
setIsTooLongDescription(ref.current.scrollHeight > 60)
```

Reply:
Done